example/android/app/src/androidTest/java/com/example/amazonivsreactnative/DetoxTest.java

```diff
@@ -1,4 +1,4 @@
-package com.example.amazonivsreactnative;
+package com.example.amazonivsreactnativeplayer;
```

Comment:
We should do the same here:
https://github.com/aws/amazon-ivs-react-native/pull/25/files#r788967093

Perhaps:

```
com.amazonaws.ivs.reactnative.player.example
```

Reply:
Actually, we should hold on this one to prevent any additional changes to the example app structure, as these are less important than the RN package itself.